drivers/test.go

```diff
@@ -74,11 +74,14 @@ func init() {
 		State:   init_spec.ConstructState(EmptyMapCid, "chain-validation"),
 	}
 
+	firstRewardState := reward_spec.ConstructState()
+	firstRewardState.LastPerEpochReward = big_spec.NewInt(100000000000000000)
```

Comment:
Use `1e9` (or whatever the right power is, I didn't count them). Or better, `big.Mul(big.NewInt(100), big.NewInt(1e18))`

Reference https://github.com/filecoin-project/specs-actors/issues/317